src/pynws/nws.py

```diff
@@ -62,6 +63,11 @@ def __init__(
         self.county_zone: Optional[str] = None
         self.fire_weather_zone: Optional[str] = None
 
+        if forecast_units in ("us", "si"):
+            self.forecast_units = forecast_units
+        else:
+            self.forecast_units = "us"
```

Comment:
If we align on this being always str, then this should raise an error.  Can slice it for different types of error if we want `TypeError`, `ValueError`, and so on, but less important.

If we allow `None`, that sets `"us"` as default, then it should be in an `elif`,  i.e. `elif forecast_units is None:` and then the else should raise an error like above.